artifactory/commands/transferfiles/status.go

```diff
@@ -0,0 +1,116 @@
+package transferfiles
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/commands/transferfiles/state"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+const sizeUnits = "KMGTPE"
+
+func ShowStatus() error {
+	var output strings.Builder
+	defer func() {
+		log.Output(output.String())
+	}()
+
+	startTimestamp, err := state.GetStartTimestamp()
+	if err != nil {
+		return err
+	}
+	if startTimestamp == 0 {
```

Comment:
Question - what happens if thr `transfer-files` command is suddenly stopped? Would the status still show as running? If this is the case, I suggest we remove this "Status" indication for bow, till we develop a way of amking it show the real running state.

Reply:
Thanks. I added "isProcessRunning" in GetLastLockTimestamp in the latest commit.